org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties

```diff
@@ -9,7 +9,7 @@
 
 NewXtextProjectWizard_WindowTitle=New Xtext Project
 WizardNewXtextProjectCreationPage_Description=Click 'Finish' to create a basic Xtext project or 'Next' to go to the advanced configuration.
-WizardNewXtextProjectCreationPage_TestingSupport=Testing Support
+WizardNewXtextProjectCreationPage_TestingSupport=Testing Support   JUnit version: 
```

Comment:
I don't think that a label with a couple of spaces is a good idea

Reply:
I was failing to display a Label. I added one, but it was not visible. Will try again.